### DISCUSSION TITLE
Core version 2.7.4 is also reported as Core:2.7.3-3-g2843a5ac

### DISCUSSION THREAD

**Discussion by ciberp** [esp8266/Arduino#7942] [Q&A] [answered]
Hi!

I am using Arduino IDE and I just installed latest version via Boards Manager with json. In my sketches I am using function `ESP.getFullVersion()` so I can check on which versions are my ESPs. Arduino IDE says that I installed version `2.7.4` but ESP function returns: `SDK:2.2.2-dev(38a443e)/Core:2.7.3-3-g2843a5ac=20703003/lwIP:STABLE-2_1_2_RELEASE/glue:1.2-30-g92add50/BearSSL:5c771be`

Is this correct or am I missing something?

Thank you for your answers and best regards,
Peter

**Comment by d-a-v** [comment, marked as answer]
That is correct and a bug from our part.
v2.7.3 and v2.7.4 are bugfix releases made by hand (mine) and are git-branched (not tagged).
The release process is still designed for reporting the last tag.
You are indeed running v2.7.4.

*edit*:

On [the release page](https://github.com/esp8266/Arduino/releases/tag/2.7.4) the git hash is the same:

> 2.7.4 - 2843a5a

as in 

> Core:2.7.3-3-g2843a5ac

which means `2.7.3` + `3` commits as `G`it hash `2843a5ac`

**Comment by ciberp** [comment]
@d-a-v thank you for this explanation.

Regards,
Peter
